### DISCUSSION TITLE
在视频页面下方的播放器图标是可以直接调用系统内的吗，为什么打不开？

### DISCUSSION THREAD

**Discussion by kimycai** [AlistGo/alist#3783] [General]
![image](https://user-images.githubusercontent.com/17998728/223985184-981f2450-fdcd-4295-97e3-dfc4e8b4c0e9.png)
比如我电脑windows里安装了VLC，但是点击VLC播放器图标似乎没有打开本地的VLC程序，是哪里设置有误吗

**Comment by anwen-anyi** [comment]
是不是安装的VLC绿色版？自己写入一下注册表是试试看 要么重现安装一下完整版让它自动写入注册表
